types/office-js-preview/index.d.ts

```diff
@@ -14376,7 +14376,7 @@ declare namespace Office {
          *
          * The add-in should use the `ewsUrl` property to determine the correct URL to use when making EWS calls.
          *
-         * You can pass both the token and either an attachment identifier or item identifier to a third-party system. The third-party system uses
+         * You can pass both the token and either an attachment identifier or item identifier to a system from Microsoft and others. That system uses
```

Comment:
Could this say "external" as well? Here and other instances.
```suggestion
         * You can pass both the token and either an attachment identifier or item identifier to an external system. That system uses
```

Reply:
@ElizabethSamuel-MSFT, I'm basing this off of @davidchesnut's guidance from this PR: https://github.com/OfficeDev/office-js-docs-pr/pull/3373, so I'll defer to him there.